### PR TITLE
Added links about tech articles for the 'Home' page

### PR DESCRIPTION
**Pull request by AnBC22**
I fetched the news and articles data from newsdata.io, just added the links to the actual articles as it was not possible to put a little description about each article because some information was available only with a paid subscription

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const feed = document.getElementById('feed')
+
+fetch('https://newsdata.io/api/1/news?apikey=pub_37020dad134a463f0666e9f1a5eba6e8b2a34&q=coding%20%20AND%20computer&language=en&category=technology')
```
Comment (michaella23) [inline main.js:3]: Do you think there is any need to hide this API key?

File: index.html [hunk 2]
```diff
@@ -26,6 +26,10 @@ <h1>Welcome to the Scrimba Bootcamp 2024!</h1>
             <img src="images/coffee.jpg" alt="A cup of coffee seen from above and with coffee beans around it" />
         </div>
         
+        <div id="feed">
```
Comment (michaella23) [inline index.html:29]: We could potentially use the semantic `<section>` element here rather than a generic `<div>`.

**Review by michaella23 — APPROVED** [review]
@AnBC22 thanks for this! Looks pretty good, and I will go ahead and merge these changes. I do think we should add a heading for this section to indicate what these links are. It might also be nice to display the `image_url` for each article. Please create issues for these to-dos and let me know if you'd like to work on them. 🙂

File: main.js (hunk 1 above)
Comment (AnBC22) [reply main.js:3]: I'm sorry, you mean we need to hide it? It was a public key given by the website so I didn't think too much about it

File: index.html (hunk 2 above)
Comment (AnBC22) [reply index.html:29]: Yes, you are right Micha! I'll create some new issues for those changes

**Pull request merged**